src/java/com/google/cloud/bigquery/dwhassessment/extractiontool/subcommand/ExtractSubcommand.java

```diff
@@ -181,7 +181,10 @@ private void scriptVars(Map<String, String> vars) {
         "The format is 'yyyy-mm-dd[Thh:mm:ss[.n...]]'. Example: '2007-12-03T10:15:30.233333'.",
         "If only the date part is specified, then time defaults to 00:00:00.00 of the given date.",
         "If --run-mode is INCREMENTAL, this will be overwritten by the timestamp of the latest"
-            + " record."
+            + " record.",
+        "Note that this also applies to sql_logs script but on a timestamp that is similar but can"
```

Comment:
I had to re-read the sentence a couple of times to fully understand its meaning and I feel it may be hard for users with not a fluent english like me. Maybe consider rephrasing it a little bit for the sake of simplicity.

E.g. `Due to Teradata design, 'sql_logs' collection timestamps may differ in seconds from 'querylogs'. Keep this in mind to ensure all the data on the edge of the desired interval is included`

Reply:
Rephrased for clarity. I didn't include the "recommendation" part because I feel the best solution depends on each user's use case.